### ISSUE TITLE
MongooseError: The `uri` parameter to `openUri()` must be a string, got "undefined". Make sure the first parameter to `mongoose.connect()` or `mongoose.createConnection()` is a string.

### ISSUE THREAD

**Issue by ilsochrist** [basir/next-tailwind-amazona#1]
![image](https://user-images.githubusercontent.com/100448527/178002782-b98773e4-aa8b-4a76-90f0-e7b7226046f3.png)

**Comment by ilsochrist** [comment]
Where are the env file? think thats why broked your code

**Comment by agbts** [comment]
Hi!  need .env file  to move to the common directory of the project,
after that, you will be able to see load .env file immediately after the "npm run dev" command

![image](https://user-images.githubusercontent.com/108715066/178207817-14da4683-02ca-41af-b2d4-bf951b45780d.png)

**Comment by ilsochrist** [comment]
yeah! i dont saw this env file in repository of github. Can you say to me where is?

**Comment by gagnol** [comment]
here! env. files never share at github ...you have to create your own credentials ... eg. mongo database , access token , etc

**Comment by Junnygram** [comment]
hello i also have this issue , how can it be solved
